tests/services/analysis_starter/test_balsamic_configurator.py

```diff
@@ -0,0 +1,439 @@
+    wgs_tumor_only_case = create_autospec(Case, data_analysis="balsamic-umi", internal_id="case_1")
+    application = create_autospec(Application, prep_category="wgs")
+    application_version = create_autospec(ApplicationVersion, application=application)
+    sample_1 = create_autospec(
+        Sample,
+        internal_id="tumor_sample",
+        is_tumour=True,
+        sex=SexOptions.MALE,
+        application_version=application_version,
+    )
+    sample_2 = create_autospec(
+        Sample,
+        internal_id="normal_sample",
+        is_tumour=False,
+        sex=SexOptions.MALE,
+        application_version=application_version,
+    )
+    wgs_tumor_only_case.samples = [sample_1, sample_2]
+    balsamic_configurator.store.get_case_by_internal_id = Mock(return_value=wgs_tumor_only_case)
+
+    # WHEN building the CLI input
+    cli_input: BalsamicConfigInput = balsamic_configurator._build_cli_input(
+        case_id=wgs_tumor_only_case.internal_id
+    )
+
+    # THEN pydantic model should be created (and validated)
+    assert isinstance(cli_input, BalsamicConfigInput)
+
+    # THEN the correct normal sample name should be set
+    assert cli_input.normal_sample_name == "normal_sample"
+
+    # THEN the correct gens_coverage_pon should be chosen (Male)
+    assert cli_input.gens_coverage_pon == balsamic_configurator.gens_coverage_male_path
+
+    # THEN fields not relevant for WGS analyses should be set to their default values
+    for field in PANEL_ONLY_FIELDS:
+        assert getattr(cli_input, field) == BalsamicConfigInput.model_fields[field].default
+
+
+def test__build_cli_input_panel_tumor_only(balsamic_configurator, bed_version_short_name):
+    # GIVEN a case with one panel sample
+    panel_tumor_only_case = create_autospec(Case, data_analysis="balsamic", internal_id="case_1")
+    application = create_autospec(Application, prep_category="tgs")
+    application_version = create_autospec(ApplicationVersion, application=application)
+    sample_1 = create_autospec(
+        Sample,
+        internal_id="sample1",
+        is_tumour=True,
+        application_version=application_version,
+        sex=SexOptions.MALE,
+    )
+    panel_tumor_only_case.samples = [sample_1]
+    balsamic_configurator.store.get_case_by_internal_id = Mock(return_value=panel_tumor_only_case)
+
+    # GIVEN that LIMS returns a panel name that exists in the store
+    balsamic_configurator.lims_api.capture_kit = Mock(return_value=bed_version_short_name)
+
+    # GIVEN that there exists matching pon files
+    balsamic_configurator._get_pon = Mock(return_value=Path("path/to/pon.cnn"))
+
+    # WHEN building the CLI input
+    cli_input: BalsamicConfigInput = balsamic_configurator._build_cli_input(
+        case_id=panel_tumor_only_case.internal_id
+    )
+
+    # THEN pydantic model should be created (and validated)
+    assert isinstance(cli_input, BalsamicConfigInput)
+
+    # THEN the correct no normal sample name should be set
```

Comment:
```suggestion
    # THEN the correct normal sample name should be set
```

Reply:
fixed in 55a9f5ae2a98a97a78b86eee6f8e13194d48b10d